src/main/java/net/javadiscord/javabot/api/routes/qotw_leaderboard/QOTWLeaderboardController.java

```diff
@@ -0,0 +1,82 @@
+package net.javadiscord.javabot.api.routes.qotw_leaderboard;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import net.dv8tion.jda.api.entities.Guild;
+import net.javadiscord.javabot.Bot;
+import net.javadiscord.javabot.api.response.ApiResponseBuilder;
+import net.javadiscord.javabot.api.response.ApiResponses;
+import net.javadiscord.javabot.api.routes.CaffeineCache;
+import net.javadiscord.javabot.api.routes.JDAEntity;
+import net.javadiscord.javabot.api.routes.qotw_leaderboard.model.QOTWMemberData;
+import net.javadiscord.javabot.systems.qotw.QOTWPointsService;
+import net.javadiscord.javabot.util.Checks;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Handles all GET-Requests on the {guild_id}/qotw/leaderboard route.
+ */
+@RestController
+public class QOTWLeaderboardController extends CaffeineCache<Long, List<QOTWMemberData>> implements JDAEntity {
+
+	/**
+	 * The constructor of this class which initializes the {@link Caffeine} cache.
+	 */
+	public QOTWLeaderboardController() {
+		super(Caffeine.newBuilder()
+				.expireAfterWrite(10, TimeUnit.MINUTES)
+				.build()
+		);
+	}
+
+	/**
+	 * Serves the specified amount of users. Sorted by the
+	 * amount of qotw-points.
+	 *
+	 * @param guildId     The guilds' id.
+	 * @param amountParam The amount of users to return. Defaults to 3.
+	 * @return The {@link ResponseEntity}.
+	 */
+	@GetMapping(
+			value = "{guild_id}/qotw/leaderboard",
+			produces = MediaType.APPLICATION_JSON_VALUE
+	)
+	public ResponseEntity<String> getQOTWLeaderboard(
```

Comment:
Spring Web already uses Jackson and is able to serialize objects with that when using `@RestController`.
If you just return a `List<QOTWMemberData>`-object (`ResponseEntity<List<QOTWMemberData>>` or `ResponseEntity<?>` if it may be one of multiple types should work as well), Spring will try to serialize that using Jackson.
With that, `produces = MediaType.APPLICATION_JSON_VALUE` might be obsolete.